apps/nextjs-website/src/components/organisms/News/NewsShowcase.tsx

```diff
@@ -5,48 +5,57 @@ import { Box } from '@mui/material';
 import Newsroom from '@/editorialComponents/Newsroom/Newsroom';
 import { useTranslations } from 'next-intl';
 
-type NewsProps = {
+type NewsShowcaseProps = {
   title: string;
   subtitle?: string;
   cta?: {
     label: string;
     href: string;
   };
   marginTop?: number;
-  cards: {
+  items: {
     comingSoon?: boolean;
     title: string;
-    dateString?: string;
-    image?: {
+    publishedAt?: string;
+    image: {
       url: string;
-      alt: string;
+      alternativeText: string | null;
     };
-    href: {
-      label: string;
-      link: string;
-      title: string;
+    link: {
+      url: string;
+      text: string;
     };
   }[];
 };
 
-const News = ({ title, subtitle, cta, marginTop, cards }: NewsProps) => {
+const NewsShowcase = ({
```

Comment:
Also Folder should be renamed accordingly `apps/nextjs-website/src/components/organisms/NewsShowcase`